langchain/src/embeddings/openai.ts

```diff
@@ -106,7 +106,12 @@ export class OpenAIEmbeddings
     this.azureOpenAIApiVersion = azureApiVersion;
     this.azureOpenAIApiKey = azureApiKey;
     this.azureOpenAIApiInstanceName = azureApiInstanceName;
-    this.azureOpenAIApiDeploymentName = azureApiDeploymentName;
+    // if openai/deployments is not in the deployment name, add it (for backwards compatibility)
+    if (azureApiDeploymentName?.indexOf("openai/deployments") === -1) {
+      this.azureOpenAIApiDeploymentName = `openai/deployments/${azureApiDeploymentName}`;
```

Comment:
I don't think this will fix. please look to my ticket https://github.com/hwchase17/langchainjs/issues/2105